### PR TITLE
Fixes a visual bug with borg frames, makes them use UpdateOverlays

### PR DESCRIPTION
**Pull request by Skeletonman0**
[OBJECTS] [BUG]
## About the PR 
This PR aims to resolve a visual issue related to borg frames where the parts would not display when picked up.
It also replaces the use of the overlays variable in update_icon with UpdateOverlays.
image

## Why's this needed? 
UpdateOverlays Good, visual bug bad

### CODE REVIEW

**Review by frawhst — APPROVED** [review]
seemsgood

File: code/obj/item/mob_parts/robot_parts.dm [hunk 1]
```diff
@@ -1067,25 +1068,32 @@ ABSTRACT_TYPE(/obj/item/parts/robot_parts/leg/right)
 			return
 
 	update_icon()
-		src.overlays = null
-		if(src.chest) src.overlays += image('icons/mob/robots.dmi', "body-" + src.chest.appearanceString, OBJ_LAYER, 2)
-		if(src.head) src.overlays += image('icons/mob/robots.dmi', "head-" + src.head.appearanceString, OBJ_LAYER, 2)
+
+		if(src.chest) src.UpdateOverlays(image('icons/mob/robots.dmi', "body-" + src.chest.appearanceString, FLOAT_LAYER, 2),"chest")
+		else src.UpdateOverlays(null,"chest")
```
Comment (ZeWaka) [inline code/obj/item/mob_parts/robot_parts.dm:1073]: while you're here, please indent these so it's not a bunch of one liners - it's a bit hard to follow with the multiple nest layers

**Review by ZeWaka — CHANGES_REQUESTED** [review]
